includes/admin/emails/class-email-setting-field.php

```diff
@@ -373,9 +373,11 @@ public static function get_recipient_setting_field( Give_Email_Notification $ema
 	public static function get_preview_setting_field( Give_Email_Notification $email, $form_id = null ) {
 		return array(
 			'name' => __( 'Preview Email', 'give' ),
-			'desc' => __(
-				'Click the "Preview Email" button to preview the email in your browser. Click the "Send Test Email" button to send a test email directly to your inbox.',
-				'give'
+			'desc' => sprintf(
```

Comment:
@kjohnson Always try to separate HTML and translation. Commit: https://github.com/impress-org/givewp/compare/ea51694cefa6...ca46d42ef2c1